assembl/static2/js/app/components/administration/voteSession/pageForm.jsx

```diff
@@ -0,0 +1,154 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n } from 'react-redux-i18n';
+import { FormGroup } from 'react-bootstrap';
+import SectionTitle from '../sectionTitle';
+import FormControlWithLabel from '../../common/formControlWithLabel';
+import FileUploader from '../../common/fileUploader';
+import TitleWithHelper from '../../common/titleWithHelper';
+import { getEntryValueForLocale } from '../../../utils/i18n';
+import {
+  updateVoteSessionPageTitle,
+  updateVoteSessionPageSubtitle,
+  updateVoteSessionHeaderImage,
+  updateVoteSessionPageInstructionsTitle,
+  updateVoteSessionPageInstructionsContent,
+  updateVoteSessionPagePropositionsTitle
+} from '../../../actions/adminActions/voteSession';
+
+type PageFormProps = {
+  headerTitle: string,
+  headerSubtitle: string,
+  headerImgUrl: string,
+  headerImgMimeType: string,
+  instructionsTitle: string,
+  instructionsContent: string,
+  propositionSectionTitle: string,
+  selectedLocale: string,
+  handleHeaderTitleChange: Function,
+  handleHeaderSubtitleChange: Function,
+  handleHeaderImageChange: Function,
+  handleInstructionsTitleChange: Function,
+  handleInstructionsContentChange: Function,
+  handlePropositionSectionTitleChange: Function
+};
+
+const PageForm = ({
+  headerTitle,
+  headerSubtitle,
+  headerImgUrl,
+  headerImgMimeType,
+  instructionsTitle,
+  instructionsContent,
+  propositionSectionTitle,
+  selectedLocale,
+  handleHeaderTitleChange,
+  handleHeaderSubtitleChange,
+  handleHeaderImageChange,
+  handleInstructionsTitleChange,
+  handleInstructionsContentChange,
+  handlePropositionSectionTitleChange
+}: PageFormProps) => {
+  const headerTitlePh = `${I18n.t('administration.ph.headerTitle')} ${selectedLocale.toUpperCase()}*`;
+  const headerSubtitlePh = `${I18n.t('administration.ph.headerSubtitle')} ${selectedLocale.toUpperCase()}`;
+  const instructionsTitlePh = `${I18n.t('administration.ph.instructionsTitle')} ${selectedLocale.toUpperCase()}*`;
+  const instructionsContentPh = `${I18n.t('administration.ph.instructionsContent')} ${selectedLocale.toUpperCase()}*`;
+  const propositionSectionTitlePh = `${I18n.t('administration.ph.propositionSectionTitle')} ${selectedLocale.toUpperCase()}*`;
+  const headerImageFieldName = 'header-image';
+  return (
+    <div className="admin-box">
+      <SectionTitle title={I18n.t('administration.voteSession.0')} annotation={I18n.t('administration.annotation')} />
+      <div className="admin-content">
+        <div className="form-container">
+          <form>
```

Comment:
maybe create distinct components for the subparts of the form ? (not sure about this one TBH)